frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -157,8 +157,15 @@ const renderChromiumUsage: CellRenderer = (
   _options,
 ) => {
   if (feature.usage?.chromium?.daily && feature.usage.chromium.daily > 0) {
-    // Format to display percentage with single decimal e.g. 0.8371 -> 83.7%.
-    return html`${(feature.usage.chromium.daily * 100).toFixed(1)}%`;
```

Comment:
Apologies for my earlier confusing comment internally. It is still a ratio between 0 and 1. I meant to say it should be at most 100% after the current logic. (whereas right now, we see values way above 100%). Once we fix the data upstream, the ratios will go back to being between 0 and 1. And we can continue with the existing logic to make it a percentage.